sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -1194,6 +1199,14 @@ createOrReplaceTableColType
     : colName=errorCapturingIdentifier dataType colDefinitionOption*
     ;
 
+colDefinitionList
+    : colDefinition (COMMA colDefinition)*
+    ;
+
+colDefinition
```

Comment:
nit: looks like we can remove `createOrReplaceTableColType` as it's completely the same as this new `colDefinition`

Reply:
Good point. Will create a follow up PR for this.